### DISCUSSION TITLE
How to config matcher in middleware to match the nested dynamic route?

### DISCUSSION THREAD

**Discussion by wukaipeng-dev** [vercel/next.js#38649] [Help]
<img width="301" alt="image" src="https://user-images.githubusercontent.com/68799055/179028659-e6605d5f-5dc3-4e8e-b657-9a03a82fb66e.png">

I set the dynamic route `[organization]` and want to execute the middleware when request page under `[organization]`. This is my code but can't work:

```javascript
export const config = {
  matcher: ["/[organization]/:path*"],
};
```

Please help !

**Comment by jvidalv** [comment]
Up
